R/ard_continuous.R

```diff
@@ -356,9 +358,13 @@ ard_continuous <- function(data,
             if (stat_name %in% c("p", "p_miss", "p_nonmiss")) {
               return(label_cards(digits = 1, scale = 100))
             }
+            if (inherits(statistic, c("Date", "POSIXct", "POSIXt"))) {
```

Comment:
Original thought did not seem to work, I think at some point the posix/underlying typeof gets lost and is only preserved as a double? Is there a way to apply the logic using `variable` based on the original dataset/not the ard object that is being created? @ddsjoberg